Makefile

```
@@ -1,17 +1,52 @@
+
+.PHONY: gen gen-check proto proto-clean proto-gen proto-fixup frb \
+        rust-test rust-check analyze check version-check
+
+# ─── Unified codegen ──────────────────────────────────────────────────
+
+## Regenerate ALL codegen outputs in correct order
+gen: proto frb
+	@echo "✅ All codegen complete"
+
+## Verify all generated code is fresh (CI + pre-push)
+gen-check: gen
```

checkmake (0.3.2)

[warning] Target body for "gen-check" exceeds allowed length of 5 lines (7).

(maxbodylength)